EpiAware/src/EpiInfModels/Renewal.jl

```diff
@@ -31,9 +31,11 @@ number.
 `initialisation_prior` for the prior distribution of ``\hat{I}_0``. The default
 `initialisation_prior` is `Normal()`.
 
-## Constructor
+## Constructors
 
-- `Renewal(; data, initialisation_prior)`.
+- `Renewal(; data, initialisation_prior)`. Construct a `Renewal` model with default update steps.
+- `Renewal(data; initialisation_prior)`. Construct a `Renewal` model with default update steps.
+- `Renewal(data, initialisation_prior, renewal_step)` Construct a `Renewal` model with `renewal_step` update step function.
```

Comment:
if its in a renewal struct does it need to be called renewal step?

Reply:
I'm interpreting this as a comment on argument naming rather than the naming of the type system? The type system is pretty verbose, but personally I like that.